File: Trabajo #5/Python/Trabajo#2.py
n1=int(input("Ingrese el numero: "))
n2=int(input("Desea ingresar otro numero? \n digite 1 si o 2 para no "))
n=2
while n2==1:
    n1=int(input("Ingrese el numero:[n1]\n numero: "))
    n2=int(input("Desea ingresar otro numero? \n digite 1 si o 2 para no "))
    n2+=1


print("El valor del cubo es: ",n1**2)
print("La cuarta parte es  : ",n1/4)
print("La quinta parte es: ",n1*5)

File: Trabajo #5/C++/Trabajo#2.cpp
#include <iostream>
#include<math.h>
using namespace std;
int n, n4, n5,o,resultado1=0,resultado2=0,resultado3=0,total1,total2,total3;
int main(){
	cout<<"Ingrese el numero: ";
	cin>>n;
	cout<<"Desea ingresar otro numero?"<<"Digite 1 si o 2 para no ";
	cin>>o;
	while (o==1){
		cout<<"Ingrese el numero: ";
		cin>>n;
		cout<<"Desea ingresar otro numero? "<<"Digite 1 si o 2 para no ";
		cin>>o;
	}
	resultado1=(pow(n,2)-1);
	total1=resultado1*resultado1;
	cout<<"El cubo es: "<<total1<<endl;
		resultado2=(pow(n,4)-1);
		total2=resultado2/4;
	cout<<"la cuarta parte es:  "<<total2<<endl;
		resultado3=(pow(n,5)-1);
		total3=resultado3/5;
	cout<<"la quinta parte es: "<<total3<<endl;
	}
